TODO

```diff
@@ -22,12 +22,12 @@ CONTENT todo
 
 1. integrate ksh into the shell chapters (assigned:paul)
 
-2. integrate dash into the shell chapters (obsolete no?)
+2. integrate dash into the shell chapters (obsolete no? its obsolete)
```

Comment:
How is Dash obsolete?

Reply:
It's not obsolete, yet I have not seen it used anywhere for  last 10 years. even Debian won't ship it with the distro itself. so should we still keep it ? btw it was like it when i clone the code.

Reply:
It's installed by default on all Debian and Ubuntu machines I know of.
Better remove the that mention about being obsolete.

Reply:
done